js/ui/html_editor/modules/tableResizing.js

```diff
@@ -380,31 +465,23 @@ export default class TableResizingModule extends BaseModule {
         const currentLineNewSize = this._startLineSize + eventOffset;
 
         if(direction === 'horizontal') {
-            const nextColumnNewSize = this._nextLineSize && this._nextLineSize - eventOffset;
-            const isCurrentColumnHasEnoughPlace = currentLineNewSize >= this._minColumnWidth;
-            const isNextColumnHasEnoughPlace = !this._nextLineSize || nextColumnNewSize >= this._minColumnWidth;
-            const $lineElements = this._getLineElements(frame.$table, index);
-            const $nextLineElements = this._getLineElements(frame.$table, index + 1);
-
-            if(isCurrentColumnHasEnoughPlace && isNextColumnHasEnoughPlace) {
-                $lineElements.each((i, element) => {
-                    $(element).attr(directionInfo.positionStyleProperty, currentLineNewSize + 'px');
-                });
-
-                if(this._nextLineSize) {
-                    $nextLineElements.each((i, element) => {
-                        $(element).attr(directionInfo.positionStyleProperty, nextColumnNewSize + 'px');
-                    });
-                }
-            }
+            this._horizontalDragHandler(currentLineNewSize, directionInfo, eventOffset, { $determinantElements, index, frame });
         } else {
-            const newHeight = Math.max(currentLineNewSize, this._minRowHeight);
-            $determinantElements.eq(index).attr(directionInfo.positionStyleProperty, newHeight + 'px');
+            this._verticalDragHandler(currentLineNewSize, directionInfo, eventOffset, { $determinantElements, index, frame });
         }
 
         this._updateFramePosition(frame.$table, frame.$frame);
     }
 
+    _dragEndHandler(options) {
+        this._$highlightedElement?.remove();
+        this._verticalDragging = false;
```

Comment:
isn't it better to set `this._verticalDragging = undefined;` here to make instance more clear?